### PR TITLE
Change mentee application instructions

### LINKED ISSUE #237: Change mentee application instructions
**Describe the bug**
- The instructions should be changed to show, how to provide the Google Doc link instead of Google drive.

**To Reproduce**
Steps to reproduce the behaviour:
- Change the description to show, how to provide the Google Doc link instead of Google drive.

**Desktop (please complete the following information):**
 - OS: macOS BigSur
 - Browser: Safari 14.0.1
 - Version: 14.0.1

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
## Purpose

The purpose of this PR is to fix #237 

## Goals

- Change the description to show, how to provide the Google Doc link instead of Google drive.

## Approach

- Change input placeholder text
- Change instruction paragraph.

## Screenshots

Screenshot 2021-08-08 at 14 23 33

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
OS: macOS BigSur
Browser: Safari 14.0.1
IDE: IntelliJ IDEA

### CODE REVIEW

File: src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx [hunk 1]
```diff
@@ -196,11 +195,11 @@ function MenteeApplication() {
                       rules={[
                         {
                           required: true,
-                          message: 'Please provide a Google Drive link!',
+                          message: 'Please provide a Google Doc link!',
                         },
                       ]}
                     >
-                      <Input placeholder={'Enter the google drive link here'} />
+                      <Input placeholder={'Enter the google doc link here'} />
```
Comment (anjula-sack) [inline src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: let's capitalize the word Google Doc

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx [hunk 2]
```diff
@@ -167,11 +167,10 @@ function MenteeApplication() {
               {isFormVisible ? (
                 <>
                   <Text className={styles.textPadding}>
-                    Download the template from the link below and make a
-                    document containing your information. Upload the document to
-                    a cloud storage platfrom (Ex: GoogleDrive, iCloud, OneDrive,
-                    etc) and provide a link. Make sure the document is in pdf
-                    and format and it is accessible to anybody with the link.
+                    Download the template from the link below, make a copy of
```
Comment (Gravewalker666) [inline src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: There is no need to download the template right? Shall we change that into something else??
@anjula-sack WDYT??

**Review by Gravewalker666 — DISMISSED** [review]

File: src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx (hunk 2 above)
Comment (Gravewalker666) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: Sorry, I meant to request changes not to approve

Comment (kumuditha-udayanga) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: The Download template button also should change, right?

Comment (kumuditha-udayanga) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: How about this
"Make a copy of the document linked below and fill out your
information. Save the google document and provide a link.
Make sure the document is accessible to anybody with the
link."

Comment (Gravewalker666) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: Yeah that's good enough

Comment (Gravewalker666) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: To what should we change the button text??

Comment (kumuditha-udayanga) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: Document Link? Document Template?

Comment (kumuditha-udayanga) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: Document Link? Document Template?

Comment (anjula-sack) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: oh yeah forgot that

Comment (anjula-sack) [reply src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: Can you provide @kumuditha-udayanga with a proper text?

**Review by anjula-sack — APPROVED** [review]
Thank you! @kumuditha-udayanga

**Pull request merged**